default.nix

```diff
@@ -72,18 +72,6 @@ rec {
     }) client server-invoker generated-purescript generate-purescript;
   };
 
-  marlowe-symbolic-lambda = pkgsMusl.callPackage ./marlowe-symbolic/lambda.nix {
```

Comment:
We can kill `pkgsMusl` and friends now as well, right? This simplifies things quite a bit, actually!

Reply:
i'd be happy to take care of that in a separate PR which should ideally a) help david focus only on the deployment parts and b) avoid increasing the PR(s) further.

That being said, @michaelpj i *am* looking forward to remove pkgsMusl ;D